doc/introduction/unsupported_gradients.rst

```diff
@@ -117,34 +117,22 @@ In particular, the following code works as expected:
 >>> print_grad()
 [-0.09983342]
 
-But the following code raises an error:
+PennyLane can now use adjoint differentation for any state based measurement on ``default.qubit``. ``lightning.qubit``
+still only supports expectation values.
 
-.. code-block:: python
+.. code-block:: python 
 
-    def print_grad_bad():
-        dev = qml.device('default.qubit', wires=1, shots=None)
+    @qml.qnode(qml.device('default.qubit'), diff_method="adjoint")
+    def circuit(x):
+        qml.IsingXX(x, wires=(0,1))
+        return qml.vn_entropy(wires=0)
 
-        @qml.qnode(dev, diff_method='adjoint')
-        def circuit(x):
-            qml.RX(x[0], wires=0)
-            return qml.state()
+>>> qml.grad(circuit)(qml.numpy.array(0.1)
+(0.2989909451499196+0j)
 
-        def cost_fn(x):
-            out = circuit(x)
-            return np.abs(out[0])
-
-        x = np.array([0.1], requires_grad=True)
-        print(qml.grad(cost_fn)(x))
-
->>> print_grad_bad()
-Traceback (most recent call last):
-  ...
-  File "C:\pennylane\pennylane\_qubit_device.py", line 951, in adjoint_jacobian
-    raise qml.QuantumFunctionError(
-pennylane.QuantumFunctionError: Adjoint differentiation method does not support measurement state
 
 Furthermore, the adjoint differentiation algorithm is analytic by nature. If the user creates a device

```

Comment:
```suggestion
Furthermore, the adjoint differentiation algorithm is analytic by nature. If a device is created
```

Minor, but I prefer not to use 'user' in user-facing docs, sounds a bit impersonal 😆